cluster_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	"strconv"
```

Comment:
I actually have a little tool for that! https://git.corp.stripe.com/vasi/gogroup

It finds a few things in sequins :(
```
> 
find . -name vendor -prune -o -name \*.go -exec gogroup -order std,other,prefix=github.com/stripe/sequins/ '{}' +

./cluster_test.go:20: Import in incorrect group at "github.com/stripe/sequins/zk/zktest"
./hdfs_sequins_test.go:13: Import in incorrect group at "github.com/stripe/sequins/backend"
./main.go:15: Import in incorrect group at "github.com/stripe/sequins/backend"
./s3_sequins_test.go:19: Import in incorrect group at "github.com/stripe/sequins/backend"
./zk/watcher_test.go:11: Import in incorrect group at "github.com/stripe/sequins/zk/zktest"
```